### ISSUE TITLE
[man page, lack of info] -p key also can be 0

### ISSUE THREAD

**Issue by MikhailKasimov** [memcached/memcached#354]
Hello!

Hope, this is the right place to put this report.

[man page, lack of info] -p key also can be 0. E.g. https://linux.die.net/man/1/memcached

```
-p <num>
    Listen on TCP port <num>, the default is port 11211. 
-U <num>
    Listen on UDP port <num>, the default is port 11211, 0 is off. 
```
'0 is off' option is also acceptable for -p parameter. E.g. -p 0 -U 11211

```
netstat -tunlp | tr -s " " | grep 11211:
udp 0 0 127.0.0.1:11211 0.0.0.0:* -
```
So, can the description to be like this:

```
-p <num>
    Listen on TCP port <num>, the default is port 11211, 0 is off. 
-U <num>
    Listen on UDP port <num>, the default is port 11211, 0 is off. 
```
?

Thanks!

**Comment by dormando** [comment]
yup, that's a good point. will fix, thanks!

**Comment by dormando** [comment]
released in 1.5.7
